integreat-compass-cli

```diff
@@ -4,13 +4,12 @@
 import sys
 from django.core.management.commands.runserver import Command as runserver
 
-
 def main():
     """Run administrative tasks."""
-    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "integreat_compass.settings")
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "integreat_compass.core.settings")
 
     # Override default port for `runserver` command
-    runserver.default_port = "8082"
+    runserver.default_port = "8082"  # TODO move to settings file
```

Comment:
Not really sure, how I can use the constand defined in a shell script in this python file or what would be the best way.

Reply:
You can just define e.g. `DEFAULT_DEV_SERVER_PORT = 8082` in `settings.py` and then do `from django.conf import settings` and access `settings.DEFAULT_DEV_SERVER_PORT` in the cli file.
Or, if you want to pass it via a bash script, you can define an environment variable in the bash script and read it in the python script via `os.environ.get("INTEGREAT_COMPASS_PORT")` or similar.

However, if you stick with the current syntax in run.sh, you can just remove the override of `runserver.default_port` in the cli completely and just rely on the command line argument:

`integreat-compass-cli runserver "localhost:${INTEGREAT_COMPASS_PORT}"`

This is also what we do for lunes-cms and integreat-cms...